src/_includes/docs/swift-package-manager/migrate-objective-c-plugin.md

```diff
@@ -0,0 +1,377 @@
+Replace `plugin_name` throughout this guide with the name of your plugin.
+The below example uses `ios`, replace `ios` with `macos`/`darwin` as applicable.
+
+1. [Enable the Swift Package Manager feature][enableSPM].
+
+2. Start by creating a directory under the `ios`, `macos`, and/or `darwin`
+   directories.
+   Name this new directory the name of the platform package.
+
+   <pre>
+   plugin_name/ios/
+   ├── ...
+   └── <b>plugin_name/</b>
+   </pre>
+
+3. Within this new directory, create the following files/directories:
+
+    - `Package.swift` (file)
+    - `Sources` (directory)
+    - `Sources/plugin_name` (directory)
+    - `Sources/plugin_name/include` (directory)
+    - `Sources/plugin_name/include/plugin_name` (directory)
+    - `Sources/plugin_name/include/plugin_name/.gitkeep` (file)
+      - This file ensures the directory is committed.
+        You can remove the `.gitkeep` file if other files are added to the
+        directory.
+
+   Your plugin should look like:
+
+   <pre>
+   plugin_name/ios/
+   ├── ...
+   └── plugin_name/
+      ├── <b>Package.swift</b>
+      └── <b>Sources/plugin_name/include/plugin_name/</b>
+         └── <b>.gitkeep</b>
+   </pre>
+
+4. Use the following template in the `Package.swift` file:
+
+   ```swift title="Package.swift"
+   // swift-tools-version: 5.9
+   // The swift-tools-version declares the minimum version of Swift required to build this package.
+   
+   import PackageDescription
+   
+   let package = Package(
+       name: "plugin_name",
+       platforms: [
+           // The platforms your plugin supports.
+           // If your plugin only supports iOS, remove `.macOS(...)`.
+           // If your plugin only supports macOS, remove `.iOS(...)`.
+           .iOS("12.0"),
+           .macOS("10.14")
+       ],
+       products: [
+           // If the plugin name contains "_", replace with "-" for the library name
+           .library(name: "plugin-name", targets: ["plugin_name"])
+       ],
+       dependencies: [],
+       targets: [
+           .target(
+               name: "plugin_name",
+               dependencies: [],
+               resources: [
+                   // If your plugin requires a privacy manifest, for example if it uses any required
```

Comment:
Required what? It feels like this is chopped off.

Reply:
`Required reason APIs` is what Apple calls these: https://developer.apple.com/documentation/bundleresources/privacy_manifest_files/describing_use_of_required_reason_api

I'll re-wrap these lines to make it clearer this is an intentional phrase and not a mistake :)

Reply:
Re-wrapped! I'll leave this open in case folks have feedback